### DISCUSSION TITLE
[resolved] Zwroty w API

### DISCUSSION THREAD

**Discussion by jakubmaguza** [allegro/allegro-api#4533] [Suggestions - resolved]
Hej,
Niestety dalej nie ma żadnego endpointu do listowania zwrotów przez API. Czy możecie podać jakąś planowaną datę wdrożenia?

**Comment by Lukasz-Zurek** [comment]
Niestety nie jesteśmy w stanie podać planowanego terminu wdrożenia.

**Comment by AureliuszBrussy** [comment]
Udostępniliśmy zasoby do obsługi zwrotów. Więcej szczegółów w [naszym newsie](https://github.com/allegro/allegro-api/issues/4780).
